Add v2.5.0 release (#616)
memgraph/docs: docs/changelog.md

sidebar_label: Changelog
---

> ## v2.5.0 - TBD

Reviewer: ```suggestion
## v2.5.0 - Dec 13, 2022
```

### Bug Fixes

> 

Reviewer: ```suggestion

- Constructors and assignment operators now work as expected, and the API type check in the `ValueNumeric` method now correctly recognizes numeric types.
- Error message support (`SetErrorMessage`) has been added to query methods that use the MAGE C++ API.
```

sidebar_label: Changelog
---

## v2.5.0 - Dec 13, 2022

### Bug Fixes

- Constructors and assignment operators now work as expected, and the API type check in the `ValueNumeric` method now correctly recognizes numeric types.
- Error message support (`SetErrorMessage`) has been added to query methods that use the MAGE C++ API.

> ### Major Features and Improvements

Reviewer: ```suggestion

### Major Features and Improvements
```

sidebar_label: Changelog
---

## v2.5.0 - TBD

### Bug Fixes

### Major Features and Improvements

> - `DISTINCT` subclause can now be used with aggregation functions.
>   Until now, if you wanted to use an aggregation function with distinct values, you had to write a query similar to this one `WITH DISTINCT n.prop as distinct_prop RETURN COUNT(distinct_prop)`. Now you can use the `DISTINCT` clause like in the following query, `RETURN COUNT(DISTINCT n.prop)` . [#654](https://github.com/memgraph/memgraph/pull/665)

Reviewer: ```suggestion
- `DISTINCT` operator can now be used with aggregation functions.
  Until now, if you wanted to use an aggregation function with distinct values, you had to write a query similar to this one `WITH DISTINCT n.prop as distinct_prop RETURN COUNT(distinct_prop)`. Now you can use the `DISTINCT` operator like in the following query, `RETURN COUNT(DISTINCT n.prop)` . [#654](https://github.com/memgraph/memgraph/pull/665)
```

> 

Reviewer: ```suggestion
- With the new configuration flag `init_file` you can execute queries from the CYPHERL file which need to be executed before the Bolt server starts and with the configuration flag `init_data_file` you can execute queries from the CYPHERL file immediately after the Bolt server starts.[#696](https://github.com/memgraph/memgraph/pull/696)

```
